src/transactions/typeTag/typeTagParser.ts

```diff
@@ -0,0 +1,246 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable class-methods-use-this */
```

Comment:
make them static?

Reply:
I should probably remove these, I believe this was copied in